clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/model/AmazonReservationReport.groovy

```diff
@@ -112,4 +139,69 @@ class AmazonReservationReport implements ReservationReport {
       return (reservedVpc.intValue() - usedVpc.intValue())
     }
   }
+
+  static class Borrow {
```

Comment:
Not sold on the name `Borrow` (I've been back and forth between Borrow, Allocation, RegionalAllocation, etc.). Happy to hear feedback either way.

The intention is to share details around how instance types are borrowed/allocated from regional reservations to cover shortfalls.

```
"borrows": [
          {
            "index": 1,
            "source": "us-west-2:*:m4.xlarge:linux",
            "sourceInstanceQuantity": 40,
            "sourceInstanceType": "m4.xlarge",
            "target": "us-west-2:c:m4.4xlarge:linux",
            "targetInstanceQuantity": 10,
            "targetInstanceType": "m4.4xlarge"
          }
        ]
```